app/src/main/resources/db/migration/V53__sak_siste_dag_som_gir_rett.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE sak
+    ADD COLUMN IF NOT EXISTS siste_dag_som_gir_rett DATE DEFAULT NULL;
+
+UPDATE sak s
+SET siste_dag_som_gir_rett = (SELECT MAX(m.til_og_med)
```

Comment:
Lurer kanskje på om du må gjøre en kode-migrering her. Denne vil ikke ta høyde for stans. Du måtte hentet siste meldeperiode per kjede.